validators/store/contract/contract_test.go

```diff
@@ -475,41 +475,48 @@ func TestContractValidatorStore_CacheChange(t *testing.T) {
 		)
 	)
 
-	testCache := func(t *testing.T, expectedCache map[uint64]validators.Validators) {
+	type testCase struct {
+		height   uint64
+		expected validators.Validators
+	}
+
+	testCache := func(t *testing.T, testCases ...testCase) {
```

Comment:
Is there any extra benefit to this function here? You could do an array of `[]testcase` with all the cases.

Reply:
I think only the difference is that the syntax is shallower in caller side.